takwimu/templates/takwimu/_includes/navbar/_dropdown.html

```diff
@@ -11,12 +11,17 @@
   <div class="container">
     <div class="row">
       <div class="col-3">
-        <h2 class="font-weight-bold">{% block dropdown_title %}Some title{% endblock %}</h2>
-        <p class="font-weight-light">{% block dropdown_desc %}Lorem ipsum{% endblock %}</p>
+        <h2 class="font-weight-bold text-right pr-3">{% block dropdown_title %}Some title{% endblock %}</h2>
```

Comment:
Please apply `.text-right` to the entire div and not each element. It's more efficient.